src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -55,7 +55,7 @@ private async Task<string> SetMultipleKeys(ConfigurationClient service, int expe
             try
             {
                 var responseGet = await service.GetAsync(batchKey);
-                key = responseGet.Result.Value;
+                key = responseGet.Value.Value;
```

Comment:
This feels awkward to me...  is this a usage pattern that you'd expect to see a lot of?  If so, then `Value` may be an awkward name choice.

Reply:
Hm, we were trying to avoid the same thing with Task<Response<T>> where it would be `task.Result.Result`.

Reply:
We need to chose what's worse here. `task.Result.Result` seems to be slightly worse because it's true independently of `T`

Reply:
Fair point.  I'd agree that `Result.Result` is more confusing.   I've not nothing to suggest off the top of my head.   `Response.Body` seems unclear, as do `Response.Payload` and `Reponse.Item` and `Response.Body`....